internal/config/default_config.go

```diff
@@ -76,6 +76,36 @@ var (
 				Name:      "endpoints.v1.",
 				PublishTo: "meshery.meshsync.core",
 			},
+			//Added endpointslice support
+			{
+				Name:      "endpointslices.v1.",
+				PublishTo: "meshery.meshsync.core",
+			},
+			//Added cronJob support
+			{
+				Name:      "cronjobs.v1.",
+				PublishTo: "meshery.meshsync.core",
+			},
+			//Added ReplicationController support
+			{
+				Name:      "replicationcontrollers.v1.",
+				PublishTo: "meshery.meshsync.core",
+			},
+			//Added storageClass support
+			{
+				Name:      "storageclasses.v1.",
+				PublishTo: "meshery.meshsync.core",
+			},
+			//Added ClusterRole support
+			{
+				Name:      "clusterroles.v1.",
+				PublishTo: "meshery.meshsync.core",
+			},
+			//Added VolumeAttachment support
+			{
+				Name:      "volumeattachments.v1.",
```

Comment:
Quick check - @Aritra8438 all of these end in .v1. with no further qualification, right?

Reply:
Hi @leecalcote, are you referring to `volumeattachments.v1.storage.k8s.io`

Considering this [review](https://github.com/meshery/meshsync/pull/219#discussion_r1297138159), I submitted this pr.

Reply:
@Aritra8438 I retract my prior guidance on that PR. I believe that is NOT the way to go, but that we need to fully match the API resource name instead.

Reply:
No problem, @leecalcote.